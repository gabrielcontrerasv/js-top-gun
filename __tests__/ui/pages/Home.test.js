import { fireEvent, render, screen } from "@testing-library/react";
import HomeComponent from "../../../components/Home/HomeComponent";

describe("HomeComponent", () => {
  test("Matches DOM Snapshot", () => {
    const wrapper = render(<HomeComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});