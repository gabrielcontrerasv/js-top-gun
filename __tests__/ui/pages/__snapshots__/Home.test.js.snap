// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeComponent Matches DOM Snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        class="col-start-1 col-end-3 row-start-1 row-span-2 sm:pl-16 md:col-start-1 md:col-end-9 md:row-start-1 md:row-end-3 overflow-hidden "
      >
        <div
          class="w-full h-full relative "
        >
          <div
            class="hidden md:block"
          >
            <span
              style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
            >
              <img
                data-nimg="fill"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
              />
              <noscript />
            </span>
          </div>
          <div
            class="md:hidden"
          >
            <span
              style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
            >
              <img
                data-nimg="fill"
                decoding="async"
                src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
                style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
              />
              <noscript />
            </span>
          </div>
        </div>
        <div
          class="mt-[80px] h-full sm:relative sm:mt-0 sm:top-[-42rem] md:flex md:justify-center md:gap-5 z-20 "
        >
          <div
            class=" h-[full] mt-[3rem] sm:mt-[4rem] top-16 row-start-1 row-end-3 md:w-[75%] sm:pl-[8rem] md:pl-5 sm:pr-10 absolute"
          >
            <h1
              class="text-[2rem] text-white text-center font-extrabold sm:text-left sm:text-4xl lg:text-6xl sm:relative md:py-5"
            >
              Welcome to Teams
              <br />
              Veterinary Clinic
            </h1>
            <div
              class="sm:w-[90%] md:w-[70%] lg:w-[60%] px-4 sm:px-0 text-justify backdrop-blur-sm "
            >
              <p
                class="font-semibold text-[#fbfbfb] tracking-wide mt-5 md:mt-[2rem] lg:text-lg "
              >
                Teams Veterinary Clinic is a full-service animal hospital and welcomes both emergency treatment cases as well as pet patients in need of routine medical, surgical, and dental care.
              </p>
              <p
                class="font-semibold text-white tracking-wide mt-[1rem] lg:text-lg "
              >
                Beyond first-rate pet care, we make our clinic comfortable, kid-friendly, and calm, so your pet can relax in the waiting room and look forward to meeting our veterinarian.
              </p>
            </div>
            <div
              class="flex gap-5 pl-4 sm:pl-0 mt-5 sm:justify-start sm:mt-[10rem] md:mt-8 "
            >
              <button
                class=" bg-dark-green text-white tracking-wider w-[8rem] h-[3rem] rounded-md hover:scale-110 duration-300"
              >
                Sign in
              </button>
              <button
                class="bg-white text-dark-green font-medium tracking-wider w-[8rem] h-[3rem] rounded-md hover:scale-110 duration-300"
              >
                Login
              </button>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      class="col-start-1 col-end-3 row-start-1 row-span-2 sm:pl-16 md:col-start-1 md:col-end-9 md:row-start-1 md:row-end-3 overflow-hidden "
    >
      <div
        class="w-full h-full relative "
      >
        <div
          class="hidden md:block"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
            />
            <noscript />
          </span>
        </div>
        <div
          class="md:hidden"
        >
          <span
            style="box-sizing: border-box; display: block; overflow: hidden; background: none; opacity: 1; border: 0px; margin: 0px; padding: 0px; position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px;"
          >
            <img
              data-nimg="fill"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position: absolute; top: 0px; left: 0px; bottom: 0px; right: 0px; box-sizing: border-box; padding: 0px; margin: auto; display: block; width: 0px; height: 0px; min-width: 100%; max-width: 100%; min-height: 100%; max-height: 100%; object-fit: cover;"
            />
            <noscript />
          </span>
        </div>
      </div>
      <div
        class="mt-[80px] h-full sm:relative sm:mt-0 sm:top-[-42rem] md:flex md:justify-center md:gap-5 z-20 "
      >
        <div
          class=" h-[full] mt-[3rem] sm:mt-[4rem] top-16 row-start-1 row-end-3 md:w-[75%] sm:pl-[8rem] md:pl-5 sm:pr-10 absolute"
        >
          <h1
            class="text-[2rem] text-white text-center font-extrabold sm:text-left sm:text-4xl lg:text-6xl sm:relative md:py-5"
          >
            Welcome to Teams
            <br />
            Veterinary Clinic
          </h1>
          <div
            class="sm:w-[90%] md:w-[70%] lg:w-[60%] px-4 sm:px-0 text-justify backdrop-blur-sm "
          >
            <p
              class="font-semibold text-[#fbfbfb] tracking-wide mt-5 md:mt-[2rem] lg:text-lg "
            >
              Teams Veterinary Clinic is a full-service animal hospital and welcomes both emergency treatment cases as well as pet patients in need of routine medical, surgical, and dental care.
            </p>
            <p
              class="font-semibold text-white tracking-wide mt-[1rem] lg:text-lg "
            >
              Beyond first-rate pet care, we make our clinic comfortable, kid-friendly, and calm, so your pet can relax in the waiting room and look forward to meeting our veterinarian.
            </p>
          </div>
          <div
            class="flex gap-5 pl-4 sm:pl-0 mt-5 sm:justify-start sm:mt-[10rem] md:mt-8 "
          >
            <button
              class=" bg-dark-green text-white tracking-wider w-[8rem] h-[3rem] rounded-md hover:scale-110 duration-300"
            >
              Sign in
            </button>
            <button
              class="bg-white text-dark-green font-medium tracking-wider w-[8rem] h-[3rem] rounded-md hover:scale-110 duration-300"
            >
              Login
            </button>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
